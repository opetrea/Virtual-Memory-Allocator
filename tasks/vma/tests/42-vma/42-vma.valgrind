==92262== Memcheck, a memory error detector
==92262== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==92262== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==92262== Command: /home/student/Desktop/tema1_sda/sd-2023-homeworks/vma
==92262== Parent PID: 92261
==92262== 
==92262== 
==92262== HEAP SUMMARY:
==92262==     in use at exit: 0 bytes in 0 blocks
==92262==   total heap usage: 11 allocs, 11 frees, 8,416 bytes allocated
==92262== 
==92262== All heap blocks were freed -- no leaks are possible
==92262== 
==92262== For lists of detected and suppressed errors, rerun with: -s
==92262== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
