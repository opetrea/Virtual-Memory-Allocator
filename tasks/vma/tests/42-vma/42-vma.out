Total memory: 0xC8 bytes
Free memory: 0x64 bytes
Number of allocated blocks: 1
Number of allocated miniblocks: 1

Block 1 begin
Zone: 0xA - 0x6E
Miniblock 1:		0xA		-		0x6E		| RWX
Block 1 end
